tests/compat.sh

```diff
@@ -142,7 +142,7 @@ print_test_case() {
 }
 
 # list_test_case lists all potential test cases in compat.sh without execution
-list_test_case() {
+list_test_cases() {
```

Comment:
Please update this in https://github.com/Mbed-TLS/mbedtls/pull/8088 as well. The spelling of the function name doesn't matter much, but if the branches diverge, it makes future backports harder.